src/Microsoft.VisualBasic/tests/UnitTests/Microsoft/VisualBasic/Logging/FileLogTraceListenerTests.cs

```diff
@@ -24,14 +24,17 @@ public void Properties()
         _ = listener.CustomLocation;
     }
 
-    [Fact]
-    public void Write()
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
```

Comment:
We should have `BoolData` available
```suggestion
    [BoolData]
```